pkg/assets/bindata.go

```diff
@@ -21395,7 +21401,7 @@ link:function(a, b) {
 var d = function() {
 c(function() {
 var c, d;
-c = d3.select(b[0]).select("text.c3-chart-arcs-title"), c && (d = a.getCenterLabelText(), c.selectAll("*").remove(), d.bigText && !d.smText ? c.text(d.bigText) :(c.insert("tspan").text(d.bigText).classed("donut-title-big-pf", !0).attr("dy", 0).attr("x", 0), c.insert("tspan").text(d.smText).classed("donut-title-small-pf", !0).attr("dy", 20).attr("x", 0)));
+c = b[0].querySelector("text.c3-chart-arcs-title"), c && (d = a.getCenterLabelText(), c.innerHTML = d.bigText && !d.smText ? d.bigText :'<tspan dy="0" x="0" class="donut-title-big-pf">' + d.bigText + '</tspan><tspan dy="20" x="0" class="donut-title-small-pf">' + d.smText + "</tspan>");
```

Comment:
shouldn't have this diff, steve hit the same issue, rebase on latest master, and clean and install your bower deps to pull the patternfly update